admin/acceptance_tests/metadatas_test.py

```diff
@@ -184,6 +184,58 @@ def test_valid_int_metadata(self, test_app, metadatas_test_data):
             '2',
             302)
 
+    def test_invalid_url_metadata(self, test_app, metadatas_test_data):
+        self._post_invalid_metadata(
+            test_app,
+            '/layers_wms/new',
+            self._base_metadata_params(metadatas_test_data),
+            '_url',
+            'gnagnagna',
+            'Must be a URL')
+
+    def test_valid_url_metadata(self, test_app, metadatas_test_data):
+        self._post_metadata(
+            test_app,
+            '/layers_wms/new',
+            self._base_metadata_params(metadatas_test_data),
+            '_url',
+            'www.111111111111111111111111111111111111111111111111111111111111.com',
+            302)
+
+    def test_invalid_json_metadata(self, test_app, metadatas_test_data):
+        self._post_invalid_metadata(
+            test_app,
+            '/layers_wms/new',
+            self._base_metadata_params(metadatas_test_data),
+            '_json',
+            '''{"colors": [{
+                "color": "black",
+                "category": "hue",
+                "type": "primary",
+                "code": {
+                    "rgba": [255,255,255,1,
+                    "hex": "#000"
+                }
+            }]}''',
+            'a parser would say "Expecting \',\' delimiter: line 7 column 26 (char 213)"')
```

Comment:
Not sure we need a translated introduction, but for the case the parser "says".
Another possibility for introduction: "Parser report: ..."